### PR TITLE
Bump Stackage to LTS Supporting GHC 8.6.4

### PR DESCRIPTION
**Pull request by iamrecursion**
### Pull Request Description
The 8.6 GHC series brings some new type-level features that will be incredibly useful for the development of the new runtime, as well as improvements to some optimisations that should be helpful in our codebase. 

It also removes the benchmark package which had bitrotted and wasn't being used.

### Important Notes
This will need to be tested:

- On Windows to check for building issues.
- With Luna Studio
- With Manager 
- With Dataframes

### Checklist
Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] The code conforms to the [Luna Style Guide](https://github.com/luna/wiki/blob/master/code-style/01.general.md).
- [x] The code has been tested where possible.

### CODE REVIEW

File: config/snapshot.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
-resolver: lts-12.26
+resolver: lts-13.12
 name: luna
 
 packages:
     - githash-0.1.3.0
+    - containers-0.5.11.0
```
Comment (mwu-tow) [inline config/snapshot.yaml]: I'd appreciate a comment explaining why we need this version pinned down. Later when I review deps list I'm not sure if I should just "bump if it compiles" or if there was some deeper reason to use older version (like changed behavior).

**Comment by mwu-tow** [comment]
We have observed severe build time increase on Windows with this PR (over 2x).
It is not clear whether GHC 8,6 is slower or if the issue has been caused by dependencies version bump.

**Comment by iamrecursion** [comment]
Closing for now as windows compilation performance is unacceptable.

**Comment by mwu-tow** [comment]
The issue apparently comes from GHC 8.6.4. When I use GHC 8.4.4 to build lts-13.12-based project, the performance is usual. Just changing the compiler version is enough to slow it down.

**Pull request closed without merging**